### ISSUE TITLE
Implementing model structure for post functionalities

### ISSUE THREAD

**Issue by GozdeUnver** [bounswe/2021SpringGroup5#275]
@salih997 and I will implement the backend functions for event and equipment posts according to the the jsons that we decided [here](https://docs.google.com/spreadsheets/d/1nTP6YCbftro9izp67gA3ic5Sz3R2mvjUxVNsyxNVQIk/edit#gid=0)

**Comment by GozdeUnver** [comment]
I created Equipment post, comment, application, badgeofferedbyevent and sport models. Wrote an abstract functional structure and its url path to describe how we will implement functions in views.py. I wrote serializers. I tested the models by makemigration and migrate commands. I pushed the branch as "feature_post" name but not created a pull request yet because it is not finished yet

**Comment by GozdeUnver** [comment]
I removed all the fields(description, equipments,rule,location and max number of players) of the sport other than the name of the sport from the sport model and decided that user should write these information on an event post by himself/herself instead of system default values because I couldn't find an api to retrieve all these information about sports. I added image path field to event and equipment post models. I created event and equipment post activity stream models which store context, summary, type and object values of an activity related to these posts. I created a skill level model to store 5 different types of skill levels. I seperated comment model into event and post comment because each has foreign keys to different post models. I created badgeofferedbyeventpost model which stores event post id and badge id tuples and badgeownedbyuser model which stores the badges owned by a user by the system or a user and when it was received.
